### ISSUE TITLE
Urgent plz! Sequence Classifier produces same output during prediction

### ISSUE THREAD

**Issue by MarwaEssam** [allenai/longformer#170]
I am trying to use the sequence classifier to do some classification for relevant or non relevant text documents given a long query text. 

The input is presented to the model like this <query tokens><s><candidate document tokens> .

I set the global attention to the CLS token, the query tokens and the separator token, and I left the classification head intact.

Now my model produces the same output for all my testing samples

here is how I train my model:

`PRE_TRAINED_MODEL_NAME = 'allenai/longformer-base-4096'
    tokenizer = LongformerTokenizer.from_pretrained(PRE_TRAINED_MODEL_NAME)
    config = LongformerConfig.from_pretrained(PRE_TRAINED_MODEL_NAME)

    config.attention_mode = 'sliding_chunks'
    config.num_labels = 2
    config.gradient_checkpointing=True
    MAX_LEN = 4096
    modeltoTune = LongformerForSequenceClassification.from_pretrained(PRE_TRAINED_MODEL_NAME, config=config)

    print(modeltoTune.config.max_position_embeddings)
    modeltoTune.train()
    training_args = TrainingArguments(
        output_dir='./results',  
        num_train_epochs=1,
        per_device_train_batch_size=1,
        per_device_eval_batch_size=1,  # batch size for evaluation
        warmup_steps=50,
        logging_dir='./logs',  # directory for storing logs
        do_train=True,
        learning_rate=0.01
    )
trainer = Trainer(
        model=modeltoTune,  # the instantiated 🤗 Transformers model to be trained
        args=training_args,  # training arguments, defined above
        train_dataset=train_data_converted
    )
    print("Done initializing trainer")
    modeltoTune.to(device)
    trainer.train()`

And here is how I test it.

`modelafterTuning = LongformerForSequenceClassification.from_pretrained('./results')
for key in test_data_converted.Labels.keys():
item = test_data_converted.__getitem__(key)
        output = modelafterTuning(input_ids=item["input_ids"].unsqueeze(0),
                                          labels=item["labels"].unsqueeze(0),
                                          attention_mask=item["attention_mask"].unsqueeze(0),
                                          global_attention_mask=item["global_attention_mask"].unsqueeze(0),
                                          return_dict=True
                                          )
        prob = torch.nn.functional.softmax(output.logits)
        print(prob)`

I tried changing the hyperparameters like for example the learning rate but nothing seems to work

**Comment by lookmyeye** [comment]
My learning rate is about 10^-6  

10^-4 won't work
